### ISSUE TITLE
Overmolding

### ISSUE THREAD

**Issue by DieKatzchen** [theelims/SmartAssPlug#1]
Would it be possible to make a variant design that could be overmolded with silicone? You'd need either a negative mold or a positive master (ideally both) and to slim down the plastic casing. This would make it MUCH easier to keep sanitary.

**Comment by theelims** [comment]
Yes, that would be the optimal solution and it's on my long term development roadmap. 

However, as this is a DIY project this creates a significant barrier to rebuild this project as working with silicone is not a widely used DIY technique. So it will always be optional. 

Protecting the plug with a condom is the preferred way.
